modules/ossm-config-openshift-monitoring.adoc

```diff
@@ -0,0 +1,136 @@
+:_mod-docs-content-type: Procedure
```

Comment:
Missing comment / metadata about where this module is used, FYI

Reply:
That is TBD at the moment. All of the assemblies/topic maps are still being worked out. This was an effort to take content from a multitude of places, and organize the entire procedure for setting up user-workload monitoring with Kiali in Service Mesh.

Reply:
I'd add in the blank comment anyway maybe with a note as a reminder to update this later 😉